gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # Commons
-kotlin = "1.6.10"
-kotlinSP = "1.6.10-1.0.4"
+kotlin = "1.8.10"
+kotlinSP = "1.8.10-1.0.9"
```

Comment:
do we need it as a part of this PR? I would rather handle it in the SDK v2 branch with Gradle 8 update (which is based on Kotlin 1.8 as well, so that we don't have conflicts for the build classpath), and also checking API/ABI compatibility.

Reply:
Unfortunately I need to use this version otherwise there's an incompatibility with the Metrics library.